test-suite/js/src/test/scala/org/scalajs/testsuite/library/WeakRefTest.scala

```diff
@@ -0,0 +1,31 @@
+/*
+ * Scala.js (https://www.scala-js.org/)
+ *
+ * Copyright EPFL.
+ *
+ * Licensed under Apache License 2.0
+ * (https://www.apache.org/licenses/LICENSE-2.0).
+ *
+ * See the NOTICE file distributed with this work for
+ * additional information regarding copyright ownership.
+ */
+
+package org.scalajs.testsuite.library
+
+import org.junit.Assert._
+import org.junit.Test
+
+import scala.scalajs.js
+
+class WeakRefTest {
+  @Test def testConstructor(): Unit = {
+    val obj = js.Object()
+    assertNotNull(new js.WeakRef[js.Object](obj))
+  }
```

Comment:
Similarly, this seems redundant with what's tested in `testDeref()`. Consider dropping it.